src/modules/simulation/simulator_ignition_bridge/SimulatorIgnitionBridge.cpp

```diff
@@ -133,13 +165,15 @@ int SimulatorIgnitionBridge::task_spawn(int argc, char *argv[])
 {
 	const char *world_name = "default";
 	const char *model_name = nullptr;
+	const char *model_pose = "0.0,0.0,0.0,0.0,0.0,0.0";
```

Comment:
We should be able to keep it completely optional.

```suggestion
	const char *model_pose = nullptr;
```